types/kafka-node-tests.ts

```diff
@@ -125,6 +125,7 @@ const cgOptions: kafka.ConsumerGroupOptions = {
 
 const consumerGroup = new kafka.ConsumerGroup(cgOptions, ['topic1']);
 consumerGroup.on('error', (err) => { });
+consumerGroup.on('connect', (err) => { });
```

Comment:
connect event does not emit with an `err`.

Reply:
Oops! resoved and send new PR